README.md

```diff
@@ -41,13 +41,46 @@
   </a>
 </p>
 
-Semgrep is a simple and flexible [command-line](#installation) and [online](https://semgrep.live/) tool for static analysis. Use pre-built or custom rules to enforce code and security standards in your codebase.
+Semgrep is a simple and flexible [command-line](#installation) and [WYSIWYG](https://semgrep.live/) tool for static analysis. Use pre-built or custom rules to enforce code and security standards in your codebase.
```

Comment:
I wouldn't use WYSIWYG. I like the changes from https://github.com/returntocorp/semgrep/pull/983/files calling it an interactive live editor